src/layouts/DocsLayout.tsx

```diff
@@ -27,6 +33,26 @@ export const DocsLayout: React.FC<Props> = ({
       `https://github.com/railwayapp/docs/edit/main/src/pages${pathname}.md`,
     [pathname],
   );
+  const [helpful, setHelpful] = useState(true);
+  const [success, setSuccess] = useState(false);
+
+  const sendFeedback = async event => {
+    event.preventDefault()
+    const feedback = event.target.feedback ? event.target.feedback.value : "";
```

Comment:
nit: Could update to
```suggestion
    const feedback = event.target.feedback ?? "";
```